README.md

```diff
@@ -5,6 +5,11 @@ scripts we use to set up some of our infrastructure at Planetary.social. These
 scripts are not designed to be used outside of Planetary, but are still
 published here in case they are useful for other scuttlebutt/nostr users.
 
+# Setup
+
+1. Install Ansible and Ansible Galaxy
+2. Run `ansible-galaxy install -r requirements.yml`
```

Comment:
I had to run this command before I could run the updated playbooks. Are these steps redundant if you use nix? Maybe we should just tell people to use that. And maybe there are more steps here, I just stubbed this out in the old README before the nix section was added.

Reply:
Yah, if you use nix+direnv, ansible and ansible-galaxy are auto-installed.   It looks like the requirements are just for the docker module, which we do want.  I believe this community module is included in the nix package we use, but will investigate further.